penthouse.gemspec

```diff
@@ -1,39 +1,39 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'penthouse/version'
+require "penthouse/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = 'penthouse'
-  spec.version       = Penthouse::VERSION
-  spec.authors       = ['Ryan Townsend']
-  spec.email         = ['ryan@ryantownsend.co.uk']
+  spec.name = "penthouse"
+  spec.version = Penthouse::VERSION
+  spec.authors = ["Ryan Townsend"]
+  spec.email = ["ryan@ryantownsend.co.uk"]
 
-  spec.summary       = %q{Multi-tenancy framework. Out of the box, supports Postgres schemas and per-tenant databases}
-  spec.homepage      = 'https://github.com/shiftcommerce/penthouse'
+  spec.summary = "Multi-tenancy framework. Out of the box, supports Postgres schemas and per-tenant databases"
+  spec.homepage = "https://github.com/shiftcommerce/penthouse"
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ['lib']
+  spec.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.bindir = "exe"
+  spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"]
 
-  spec.add_dependency 'bundler', '~> 1.11'
+  spec.add_dependency "bundler", ">= 1.11"
 
-  spec.add_development_dependency 'bundler', '~> 1.11'
-  spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'yard', '~> 0.8.7.6'
+  spec.add_development_dependency "bundler", ">= 1.11"
+  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "yard", "~> 0.8.7.6"
   # testing
-  spec.add_development_dependency 'rspec', '~> 3.4.0'
-  spec.add_development_dependency 'simplecov', '~> 0.11.2'
-  spec.add_development_dependency 'rubocop', '~> 0.38'
+  spec.add_development_dependency "rspec", "~> 3.4.0"
+  spec.add_development_dependency "simplecov", "~> 0.11.2"
+  spec.add_development_dependency "rubocop", "~> 0.38"
+  spec.add_development_dependency "standardrb"
   # web
-  spec.add_development_dependency 'rack', '~> 1.6.4'
+  spec.add_development_dependency "rack", "~> 1.6.4"
   # db
-  spec.add_development_dependency 'ar-octopus', '~> 0.8.6'
-  spec.add_development_dependency 'activesupport', '~> 4.2.6'
-  spec.add_development_dependency 'activerecord', '~> 4.2.6'
-  spec.add_development_dependency 'pg', '~> 0.21.0'
+  spec.add_development_dependency "ar-octopus", "~> 0.10.2"
```

Comment:
@ryantownsend  this is the only change.

See the release https://github.com/thiagopradi/octopus/commit/1c99859dc5e46aefc62573db7ac80e2a14eedb11

Our version lock: https://github.com/shiftcommerce/flex-platform/blob/4e1e9040e31e95c8975ec23acc4c0e014feb5f1d/Gemfile#L38-L42

Looks like v0.9.2 could be an option aswell

Reply:
Might make sense to perform the platform update first; then perform the Penthouse update